src/homework4/task2.py

```diff
@@ -0,0 +1,32 @@
+"""Города
+   Дан список стран и городов каждой страны. Затем даны названия городов.
+   Для каждого города укажите, в какой стране он находится.
+   Входные данные
+   Программа получает на вход кол-во стран N. Далее идет N строк,каждая строка
+   начинается с названия страны, затем идут названия городов этой страны.
+   В следующей строке записано число M, далее идут M запросов — названия
+   каких-то M городов, перечисленных выше.
+   Выходные данные
+   Для каждого из запроса выведите название страны,
+   в котором находится данный город.
+"""
+
+
+dct_ = dict()
+ctr_ = []
+ct_ = []
+N = int(input())
+for i in range(N):
+    a = input()
+    b = a.split()
+    ctr_.insert(len(ctr_), b[0])
+    ct_.insert(len(ct_), b[1: len(b)])
+
+dct_ = dict(zip(ctr_, ct_))
```

Comment:
unclear final dict structure because of naming :(

but it seems it would be {country: cities} - it's not optimal structure for this task (please take a look at next comment)